scripts/factories/photographer.js

```diff
@@ -1,17 +1,73 @@
-function photographerFactory(data) {
-    const { name, portrait } = data;
-
-    const picture = `assets/photographers/${portrait}`;
-
-    function getUserCardDOM() {
-        const article = document.createElement( 'article' );
-        const img = document.createElement( 'img' );
-        img.setAttribute("src", picture)
-        const h2 = document.createElement( 'h2' );
-        h2.textContent = name;
-        article.appendChild(img);
-        article.appendChild(h2);
-        return (article);
-    }
-    return { name, picture, getUserCardDOM }
-}
\ No newline at end of file
+function photographerFactory(data) {  // eslint-disable-line no-unused-vars
+	const { name, portrait, city, country, tagline, price, id } = data
+	const picture = `assets/photographers/${portrait}`
+	const link = `photographer.html?id=${id}`
+
+	function getUserCardDOM() {
+		const article = document.createElement( "article" )
+		const img = document.createElement( "img" )
+		const h2 = document.createElement( "h2" )
+		const location = document.createElement("p")
+		const pTagline = document.createElement("p")
+		const pPrice = document.createElement("p")
+		const a = document.createElement("a")
+		const alt = "photo de" + name
+
+		img.setAttribute("src", picture)
+		img.setAttribute("alt", alt)
+		article.setAttribute("class", "photographer")
+		location.setAttribute("class", "location")
+		pTagline.setAttribute("class", "tagline")
+		pPrice.setAttribute("class", "price")
+		a.setAttribute("href", link)
+
+		h2.textContent = name
+		location.textContent = city + ", " + country
+		pTagline.textContent = tagline
+		pPrice.textContent = price + "€/jour"
+
+		article.appendChild(a)
+		a.appendChild(img)
+		a.appendChild(h2)
+		article.appendChild(location)
+		article.appendChild(pTagline)
+		article.appendChild(pPrice)
+
+		return (article)
```

Comment:
```suggestion
		return article;
```

D'ailleurs il n'y a de ";" virgules nulle pas sur les fichiers, au secours 😱  **Prettier**!!!!